app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Category Button -->
+    <dimen name="category_button_width">0dp</dimen>
+    <dimen name="category_button_height">104dp</dimen>
+    <dimen name="category_button_margin">8dp</dimen>
+    <dimen name="category_button_weight">1</dimen>
+    <dimen name="category_button_text_size">20sp</dimen>
+
+    <!-- Speech Button -->
+    <dimen name="speech_button_text_size">24sp</dimen>
+    <dimen name="speech_button_width">0dp</dimen>
+    <dimen name="speech_button_height">128dp</dimen>
+    <dimen name="speech_button_margin">10dp</dimen>
+    <dimen name="speech_button_weight">1</dimen>
+
+    <!-- Pause Button -->
+    <dimen name="pause_button_margin_start">8dp</dimen>
+    <dimen name="pause_button_margin_top">8dp</dimen>
+    <dimen name="pause_button_text_size">20sp</dimen>
+
+
+    <!-- Settings Button -->
+    <dimen name="settings_button_margin_top">8dp</dimen>
```

Comment:
If you have duplicate `dp` values for certain dimensions, you can combine them into one, like `<dimen name="settings_button_margin">8dp</dimen>`. Saves you from having to get too granular.